From akpm@osdl.org Thu Apr 27 02:31:41 2006
Message-Id: <200604270930.k3R9UkoG025245@shell0.pdx.osdl.net>
Subject: [patch 1/9] drivers/w1/w1.c: fix a compile error
To: greg@kroah.com
Cc: akpm@osdl.org, bunk@stusta.de, johnpol@2ka.mipt.ru
From: akpm@osdl.org
Date: Thu, 27 Apr 2006 02:29:18 -0700


From: Adrian Bunk <bunk@stusta.de>

drivers/w1/w1.c:197: error: static declaration of 'w1_bus_type' follows non-static declaration
drivers/w1/w1.h:217: error: previous declaration of 'w1_bus_type' was here

Signed-off-by: Adrian Bunk <bunk@stusta.de>
Cc: Evgeniy Polyakov <johnpol@2ka.mipt.ru>
Signed-off-by: Andrew Morton <akpm@osdl.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/w1/w1.h |    1 -
 1 file changed, 1 deletion(-)

--- gregkh-2.6.orig/drivers/w1/w1.h
+++ gregkh-2.6/drivers/w1/w1.h
@@ -211,7 +211,6 @@ static inline struct w1_master* dev_to_w
 }
 
 extern struct device_driver w1_master_driver;
-extern struct bus_type w1_bus_type;
 extern struct device w1_master_device;
 extern int w1_max_slave_count;
 extern int w1_max_slave_ttl;
